### PR TITLE
Add Prague PostgreSQL November Meetup

### PR DESCRIPTION
**Pull request by gulcin**
## Summary

-

### Blog Release Checklist

- \[x] Release on: \<mm/dd/yy>
- \[ ] Published on dev.to (connected to Xata org)
- \[ ] OG image included
- \[ ] Social media is scheduled for this post

### CODE REVIEW

File: prague-postgres-meetup-nov-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
+title: 'Prague PostgreSQL Meetup on November 18th'
+description: 'The Prague PostgreSQL Meetup will take place on November 18th!'
```
Comment (alexfrancoeur) [inline prague-postgres-meetup-nov-2024.mdx:3]:
```suggestion
description: 'The Prague PostgreSQL meetup will take place on November 18th'
```

File: prague-postgres-meetup-nov-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    'p2d2'
+  ]
+published: true
+slug: prague-postgres-meetup-november-2024
+ogImage: https://us-east-1.storage.xata.sh/uq2d57njp620sncl5uvdkckvu6a9qr7k
+---
+
+## Prague PostgreSQL Meetup on November 18th
+
+I am happy to announce that the next [Prague PostgreSQL Meetup](https://www.meetup.com/prague-postgresql-meetup/events/303386799) will take place on Monday, November 18th at 17:30 at [Spaces, Nile House](https://www.spacesworks.com/cz/prague-cz/nile-house/) (Karolinská 654/2, 186 00 Praha 8-Rohanský ostrov).
```
Comment (alexfrancoeur) [inline prague-postgres-meetup-nov-2024.mdx:31]: if this is going live tomorrow, we could say "today" in addition to the actual date

**Review by alexfrancoeur — APPROVED** [review]
Very small suggestions, but LGTM ✅

**Comment by gulcin** [comment]
> Very small suggestions, but LGTM ✅

Thanks @alexfrancoeur merging it now!

**Pull request merged**